lib/hello_web/views/layout_view.ex

```diff
@@ -0,0 +1,12 @@
+defmodule HelloWeb.LayoutView do
```

Comment:
@keliumJU how does a view work?

Reply:
The views allow us to add extra functionality to the templates (heex files), since the templates are compiled inside the views we can create functions and call them inside our heex files, without the need to go through the controller, that way we separate the logic for UI-only tasks.